Traceback (most recent call last):
  File "main.py", line 3, in <module>
    from WellKnownHandler import WellKnownHandler
ModuleNotFoundError: No module named 'WellKnownHandler'
Client found in config, using: 5433d12f-472c-46ff-8878-7af38d024dc1
UMA Handler created, with control over 1 resources: ['17ac124f-7352-4283-9ced-97ebb2e5cec6']
-----------STATUS-----------
1 Locally cached resources, with IDS: ['17ac124f-7352-4283-9ced-97ebb2e5cec6']
1 Actual Resources registered in the AS, with IDS: ['17ac124f-7352-4283-9ced-97ebb2e5cec6']
-----------LIVE INFORMATION FROM AS------
{'_id': '17ac124f-7352-4283-9ced-97ebb2e5cec6', 'name': 'ADES Service', 'icon_uri': '/ADES', 'resource_scopes': ['Authenticated'], 'scope_expression': None, 'description': None, 'type': None, 'iat': 1591112408, 'exp': 1593704408}
++++++++++++++++
-----------STATUS END-------
Resource already existed, moving on
 * Serving Flask app "main" (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: on
Traceback (most recent call last):
  File "main.py", line 116, in <module>
    host=g_config["service_host"]
  File "/home/vagrant/um-pep-engine/tests/env/lib/python3.6/site-packages/flask/app.py", line 990, in run
    run_simple(host, port, self, **options)
  File "/home/vagrant/um-pep-engine/tests/env/lib/python3.6/site-packages/werkzeug/serving.py", line 1030, in run_simple
    s.bind(server_address)
OSError: [Errno 98] Address already in use
NOTICE: Client not found, generating one... 
NEW CLIENT created with ID '56b1e803-9202-4392-b5aa-79cc82067f94', since no client config was found on config.json
New client saved to config!
UMA Handler created, with control over 0 resources: []
-----------STATUS-----------
0 Locally cached resources, with IDS: []
0 Actual Resources registered in the AS, with IDS: []
-----------LIVE INFORMATION FROM AS------
-----------STATUS END-------
Created resource 'ADES Service' with ID :38371be8-68c2-48cf-a55b-a91abfb12444
 * Serving Flask app "main" (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: on
INFO:werkzeug: * Running on http://0.0.0.0:5566/ (Press CTRL+C to quit)
INFO:werkzeug: * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 307-309-544
